dashboards-observability/public/components/visualizations/charts/__tests__/horizontal_bar.test.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import { waitFor } from '@testing-library/react';
+import { Bar } from '../bar/bar';
+import {
+  LAYOUT_CONFIG,
+  HORIZONTAL_BAR_TEST_VISUALIZATIONS_DATA,
+} from '../../../../../test/event_analytics_constants';
+
+describe('Bar component', () => {
+  configure({ adapter: new Adapter() });
+
+  it('Renders bar component', async () => {
```

Comment:
Do we need to change it to corresponding name 'horizontal bar' in this case?

Reply:
done